client/src/utils/systems.js

```diff
@@ -42,12 +42,12 @@ export function findSystemDisplayName(systemList, system, isRoot) {
  * @return {string} project title
  */
 
-export function findProjectTitle(projectsList, projectSystem) {
+export function findProjectTitle(projectsList, projectSystem, projectTitle) {
   const matching = projectsList.find(project => project.id === projectSystem);
   if (matching) {
     return matching.description;
   }
-  return '';
+  return projectSystem ? projectTitle : '';
```

Comment:
This works but is non-intuitive. Maybe if projectTitle was renamed "default project title"? It seems like I have to have a projectTitle in hand before I can search for one. Can the responsibility of falling back to a default project title be given to the caller of this function instead?

Reply:
👍  on `defaultProjectTitle` or `fallbackProjectTitle`

Reply:
Hmm it should be the same `projectTitle` value though, one is just faster to look up